tasmota/berry/drivers/Core200S.be

```diff
@@ -0,0 +1,712 @@
+import string
+import webserver
+
+class StateEnum
+    var value_map
+    var value
+
+    def init(value_map, value)
```

Comment:
Please add comments to give a hint about parameters, and what the class is suppose to do